docs/_config.yml

```
@@ -17,18 +17,18 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
-    - 'https://docs.lakefs.io/'
   'Blog':
     - 'https://lakefs.io/blog/'
+  'Guides':
+    - 'https://lakefs.io/guides/'  
   'Community':
-    - 'https://lakefs.io/community/'
+    - 'https://lakefs.io/developers/'
```

Can we leave this unchanged please? 

```suggestion
    - 'https://lakefs.io/community/'
```